### PR TITLE
Use node version from nvm config for bundle-size action

### PR DESCRIPTION
**Pull request by swissspidy**
Might help prevent flakiness of this build job.

Background: this project uses `.nvmrc` to specify the Node version that this project works best with and has been tested against. It ensures that everyone runs the project with the same Node version via Node Version Manager (nvm).

Now, on a CI environment the default Node version might change, and might not be the same as the one we define in `.nvmrc`. This can cause issues, for example when installing modules with binaries that were compiled against a different Node version.

It is important that all tools work with the same node version.

This PR makes sure the compressed-size-action uses the same Node version as all other workflows and aspects of the project: the one specified in `.nvmrc`.

**tl;dr**: When the bundle-size workflow runs and executes `npm install`, it will now use whatever Node version we specifcy in `.nvmrc` rather than using the CI's default Node version.

### CODE REVIEW

**Comment by spacedmonkey** [comment]
Can you explain a little what this PR does?

**Comment by swissspidy** [comment]
@spacedmonkey Updated the description.

**Review by spacedmonkey — APPROVED** [review]
LTGM
